### PR TITLE
Some updates in Lighthouse Book

### PR DESCRIPTION
**Pull request by chong-he**
Some updates based on issues seen on the Discord server.

### CODE REVIEW

File: book/src/ui-configuration.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ To enable the HTTP API for the beacon node, utilize the `--gui` CLI flag. This a
 
 If you require accessibility from another machine within the network, configure the `--http-address` to match the local LAN IP of the system running the Beacon Node and Validator Client.
 
-> To access from another machine on the same network (192.168.0.200) set the Beacon Node and Validator Client `--http-address` as `192.168.0.200`.
+> To access from another machine on the same network (192.168.0.200) set the Beacon Node and Validator Client `--http-address` as `192.168.0.200`. When this is set, the validator client requires the flag `--beacon-nodes http://192.168.0.200:5052` to connect tot he beacon node.
```
Comment (michaelsproul) [inline book/src/ui-configuration.md]:
```suggestion
> To access from another machine on the same network (192.168.0.200) set the Beacon Node and Validator Client `--http-address` as `192.168.0.200`. When this is set, the validator client requires the flag `--beacon-nodes http://192.168.0.200:5052` to connect to the beacon node. 
```

File: book/src/ui-faqs.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ The required Api token may be found in the default data directory of the validat
...
 
 ## 4. How do I connect Siren to Lighthouse from a different computer on the same network?
-The most effective approach to enable access for a local network computer to Lighthouse's HTTP API ports is by configuring the `--http-address` to match the local LAN IP of the system running the beacon node and validator client. For instance, if the said node operates at `192.168.0.200`, this IP can be specified using the `--http-address` parameter as `--http-address 192.168.0.200`.
+The most effective approach to enable access for a local network computer to Lighthouse's HTTP API ports is by configuring the `--http-address` to match the local LAN IP of the system running the beacon node and validator client. For instance, if the said node operates at `192.168.0.200`, this IP can be specified using the `--http-address` parameter as `--http-address 192.168.0.200`. When this is set, the validator client requires the flag `--beacon-nodes http://192.168.0.200:5052` to connect tot he beacon node.
```
Comment (michaelsproul) [inline book/src/ui-faqs.md]:
```suggestion
The most effective approach to enable access for a local network computer to Lighthouse's HTTP API ports is by configuring the `--http-address` to match the local LAN IP of the system running the beacon node and validator client. For instance, if the said node operates at `192.168.0.200`, this IP can be specified using the `--http-address` parameter as `--http-address 192.168.0.200`. When this is set, the validator client requires the flag `--beacon-nodes http://192.168.0.200:5052` to connect to the beacon node. 
```

**Review by michaelsproul — APPROVED** [review]
Nice, just noticed 2 typos then we can merge

**Comment by AgeManning** [comment]
@mergifyio queue

**Pull request merged**